corehq/apps/translations/app_translations.py

```diff
@@ -191,29 +196,63 @@ def process_bulk_app_translation_upload(app, workbook):
     return msgs
 
 
-def validate_bulk_app_translation_upload(app, workbook, email):
+def run_translation_checker(file_obj):
+    translation_checker_messages = []
+    result_wb = None
+    try:
+        result_wb = validate_workbook(openpyxl.load_workbook(file_obj), translation_checker_messages)
```

Comment:
validate_workbook comes from the library,and the library requires openpyxl. would it be possible to just provide a file object and it loads the workbook for you?

Reply:
https://github.com/dimagi/CommcareTranslationChecker/pull/16